src/test/java/ru/training/at/hw2/ex2/ContentOfTheServicePageTest.java

```diff
@@ -0,0 +1,102 @@
+package ru.training.at.hw2.ex2;
+
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.Test;
+import ru.training.at.hw2.GeneralConditions;
+import ru.training.at.hw2.TestDataProvider;
+
+
+public class ContentOfTheServicePageTest extends GeneralConditions {
+
+    //5. Open through the header menu Service -> Different Elements Page
+    @Test(priority = 2)
+    public void openServicePage() {
+        driver.findElement(By.xpath("//a[contains(text(), 'Service')]")).click();
+        driver.findElement(By.xpath("//a[contains(text(), 'Different elements')]")).click();
+        softAssert.assertEquals(driver.getTitle(), "Different Elements");
+        softAssert.assertAll();
+    }
+
+    // 6. Select checkboxes
+    @Test(priority = 3)
+    public void selectWaterAndWindCheckboxes() {
+        List<WebElement> checkboxes = driver.findElements(By.cssSelector("input[type='checkbox']"));
+        WebElement water = null;
+        WebElement wind = null;
```

Comment:
Why do you set these elements to null here?

Reply:
otherwise elements may not be initialized.